src/JPNIC.xml

```diff
@@ -0,0 +1,36 @@
+    </crossRefs>
+    <text>
+      <titleText>
+        <p>Japan Network Information Center License</p>
+      </titleText>
+      <copyrightText>
+        <p>Copyright (c) <alt name="copyright" match=".+">2000-2002 Japan Network Information Center</alt>. All rights reserved.</p>
+      </copyrightText>
+      <p>By using this file, you agree to the terms and conditions set forth bellow.</p>
+      <p>LICENSE TERMS AND CONDITIONS</p>
+      <p>The following License Terms and Conditions apply, unless a different license is obtained from <alt match=".+" name="organizationClause1">Japan Network Information Center</alt> ("<alt match=".+" name="organizationClause2">JPNIC</alt>"), <alt match=".+" name="organizationClause3">a Japanese association, Kokusai-Kougyou-Kanda Bldg 6F, 2-3-4 Uchi-Kanda, Chiyoda-ku, Tokyo 101-0047, Japan</alt>.</p>
+      <list>
+        <item><bullet>1.</bullet>
+          Use, Modification and Redistribution (including distribution of any modified or derived work) in source and/or binary forms is permitted under this License Terms and Conditions.
+        </item>
+        <item><bullet>2.</bullet>
+          Redistribution of source code must retain the copyright notices as they appear in each source code file, this License Terms and Conditions.
+        </item>
+        <item><bullet>3.</bullet>
+          Redistribution in binary form must reproduce the Copyright Notice, this License Terms and Conditions, in the documentation and/or other materials provided with the distribution. For the purposes of binary distribution the "Copyright Notice" refers to the following language: "Copyright (c) <alt name="copyright" match=".+">2000-2002 Japan Network Information Center</alt>. All rights reserved."
+        </item>
+        <item><bullet>4.</bullet>
+          The name of <alt match=".+" name="organizationClause2">JPNIC</alt> may not be used to endorse or promote products derived from this Software without specific prior written approval of <alt match=".+" name="organizationClause2">JPNIC</alt>.
+        </item>
+        <item><bullet>5.</bullet>
+          Disclaimer/Limitation of Liability: THIS SOFTWARE IS PROVIDED BY <alt match=".+" name="organizationClause2">JPNIC</alt> "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL <alt match=".+" name="organizationClause2">JPNIC</alt> BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.
```

Comment:
Something I'm not sure about -- in other SPDX templates, I've seen each `name` attribute be different for `alt` tags, even where they're likely the same text. E.g., you might want to change `organizationClause2` to be `organizationClause4`, `organizationClause5`, etc., incrementing each time.

That's because sometimes the text would be slightly different in these fields. E.g., for the first one someone might put "THE COPYRIGHT HOLDERS **AND** CONTRIBUTORS" but for the second one "THE COPYRIGHT HOLDERS **OR** CONTRIBUTORS". And for SPDX matching purposes we'd want to still treat that as matching the `JPNIC` license.

@goneall, I'd welcome your input here in case I'm off-base.  =)

Reply:
@swinslow You are correct in making the names unique for each alt tag.  That is a convention we have been following for the other licenses so that we can uniquely refer to each of the fields.

Reply:
This license defines a term, "JPNIC", and reuses it throughout its text.

Reply:
True, in the given instance[1] of the license, they reuse the same term. But we would still want to match if the proper nouns (as listed in found/scanned source) were each different from one another, if they had set those differently in other instances of the license in source in the wild.

I had actually thought it was a **requirement** that the name of &lt;alt match=&gt; fields were each unique, but now I don't recall seeing this mentioned, so it could just be convention. But even if only convention, I think we should stick with it to be consistent with other licenses in the repository.

This change, and pretty-printing (or some kind of line wrapping) are the only things I would suggest with this PR. Great job @kemitchell, and thanks for taking this on!

[1] https://gitlab.isc.org/isc-projects/bind9/blob/master/COPYRIGHT#L366